src/EnergyPlus/Furnaces.cc

```diff
@@ -4524,26 +4516,15 @@ namespace Furnaces {
             }
             // Set the heat pump cooling coil convergence
             thisFurnace.CoolingConvergenceTolerance = Numbers(3);
-            // Set the heatpump cycling rate
-            thisFurnace.MaxONOFFCyclesperHour = Numbers(4);
-
-            // Set the heat pump time constant
-            thisFurnace.HPTimeConstant = Numbers(5);
-
-            // Set the heat pump on-cycle power use fraction
-            thisFurnace.OnCyclePowerFraction = Numbers(6);
-
-            // Set the heat pump fan delay time
-            thisFurnace.FanDelayTime = Numbers(7);
```

Comment:
Don't need to set parent object latent degradation inputs, since they are now part of the child coils.